ex2_hal/adcs/equipment_handler/inc/adcs_handler.h

```diff
@@ -481,6 +481,14 @@ typedef struct {
     uint16_t crc16_checksum;
 } adcs_file_info;
 
+typedef struct __attribute__((packed)){
+    uint8_t type;
+    uint8_t counter;
+    uint32_t size;
+    char file_name[20];
+} adcs_file_download_id;
+
```

Comment:
Doesn't look like clang-format was run on this. I think it would have caught the extra whitespace